packages/eslint-plugin-pf-codemods/lib/rules/v5/react-icons-warn-remove-props.js

```diff
@@ -0,0 +1,34 @@
+const {getPackageImports} = require("../../helpers");
+
+// https://github.com/patternfly/patternfly-react/pull/5275
+module.exports = {
+  meta: {fixable: "code"},
+  create: function (context) {
+    const imports = getPackageImports(context, "@patternfly/react-icons");
+
+    const iconImport = imports.filter((specifier) =>
+      specifier.imported.name.includes("Icon")
+    );
+    return imports.length === 0
+      ? {}
+      : {
+          JSXOpeningElement(node) {
+            if (
```

Comment:
let's also check that `node.name.name === iconImport.local.name`, otherwise this could match on anything that has a size/color/noVerticalAlign attribute as long as there is a pf icon import.